### PR TITLE
add --to-mp4 argument to resize_videos.py

### PR DESCRIPTION
**Pull request by MakeCent**
## Motivation

Current ``resize_videos.py`` doesn't support resizing **.webm** videos. 

1. It can be solved by change the ``-c:v libx264`` to ``-c:v libvpx``: https://github.com/open-mmlab/mmaction2/blob/cf15d56ca36ef5ad96463503f3935d8bb0b4c726/tools/data/resize_videos.py#L34

2. Or just outputting **.mp4** videos, which is what this PR does. 

I choose the second solution because I found writing **.mp4** files saved lots of time. BTW, it requires lots of **extra** time to write the resized videos (which run in the background) **after** the python script execution is finished.

## Modification

Add ``--to-mp4`` argument, to output .mp4 files.

## Others

I had tested it on the ActivityNet dataset, which contains .mkv, .webm abd .mp4 videos.

### CODE REVIEW

File: tools/data/resize_videos.py [hunk 1]
```diff
@@ -17,6 +17,10 @@ def resize_videos(vid_item):
         bool: Whether generate video cache successfully.
     """
     full_path, vid_path = vid_item
+    if args.to_mp4:
```
Comment (dreamerlin) [inline tools/data/resize_videos.py:20]: Need a comment to explain these codes lines

Comment (MakeCent) [reply tools/data/resize_videos.py:21]: Done

**Comment by kennymckormick** [comment]
@dreamerlin Better to have lint passed before merging.

**Comment by makecent** [comment]
> @dreamerlin Better to have lint passed before merging.

My bad. I used the Pycharm to run commit. Do you know how to enable pre-commit in Pycharm?

**Comment by dreamerlin** [comment]
run `pre-commit run --all-files` in command line

**Comment by makecent** [comment]
> run `pre-commit run --all-files` in command line

Sorry to bother you. But after I installed the pre-commit following the instructions provided by the mmcv, when I run ``pre-commit run --all-files``, an error occurs:
```
[INFO] Installing environment for https://github.com/jumanjihouse/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('gem', 'build', 'fake_gem__.gemspec')
return code: 1
expected return code: 0
stdout:
    Executable `gem` not found
stderr: (none)
Check the log at /home/louis/.cache/pre-commit/pre-commit.log
```
I had tested it on two different computers and both encounter this error. Any suggestion?

**Comment by dreamerlin** [comment]
ref: https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md

**Comment by makecent** [comment]
> ref: https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md

Thx. I solved it by ``sudo apt install ruby``. A new pull request #1026 was created to solve the format issue.

**Comment by OpenMMLab-Q** [comment]
Dear makecent,

First of all, we want to express our gratitude for your significant PR in the MMAction2 project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

If you are Chinese or have **WeChat**，welcome to join our community on WeChat. You can add our assistant ：**openmmlabwx**. Please add "**mmsig + Github ID**" as a remark when adding friends：）

We would also like to invite you to join our Special Interest Group (SIG) private channel on **Discord**, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. We look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

Thank you again for your contribution❤

Best regards！@makecent

**Pull request merged**